packages/runner/lib/client.unit.test.ts

```diff
@@ -46,14 +47,8 @@ describe('Runner client', () => {
         `;
         const isInvokedImmediately = false;
         const isWebhook = false;
-        const run = client.run.mutate({ nangoProps, isInvokedImmediately, isWebhook, code: jsCode });
-        console.log(run);
-        // TODO
-        //await expect(run).resolves.toEqual([1, 2, 3]);
-    });
 
-    it('should cancel a sync', async () => {
-        const result = await client.cancel.mutate({ syncId: 'sync-id' });
-        expect(result).toEqual({ ok: true, res: 'cancelled' });
+        const run = await exec(nangoProps, isInvokedImmediately, isWebhook, jsCode);
+        expect(run).toEqual([1, 2, 3]);
```

Comment:
keep a TODO to revert back to testing the entire call to `runner.run` but otherwise great

Reply:
Thanks, added with https://github.com/NangoHQ/nango/pull/1662/commits/77fb37ab0ef15d8728f8564b2c7f3422c87c4a55